apps/core/templates/news/news_list.html

```diff
@@ -19,6 +21,16 @@
         {% endif %}
         <h2>News</h2>
         <div class="row row-cols-1 row-cols-md-3 g-2 mt-2" id="NewsGrid">
+            <div class="col" id="NewsletterCard">
+                <div class="card h-100 border border-info">
+                    <img class="card-img-top mt-1" style="height: 17.5vh; object-fit: contain;" src="{% static 'cosmos/img/cosmos_alternate_grey.svg' %}">
+                    <div class="card-body">
+                        <h5 class="card-title">Subscribe to our newsletter!</h5>
+                        <p>Keep up with everything that happens at Cosmos and more!</p>
+                        <a class="stretched-link" href="{% url 'cosmos_users:subscribe-newsletter' active_tab='preferences'%}"></a>
+                    </div>
+                </div>
+            </div>
```

Comment:
I didn't really know how to make the button look/feel nice on the page so that's why I went with a card, as it also grabs the attention of the viewer more, but I can understand the issue it might cause on mobile so I can try to just make it a button if you want.